<launch>
	<node pkg="rc_car" type="navfn_stand" name="rccar_navfn">
		<rosparam file="$(find rc_car)/config/move_base/move_base.yaml" command="load"/>
		<rosparam file="$(find rc_car)/config/move_base/costmap_common.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find rc_car)/config/move_base/costmap_global.yaml" command="load"  />
		
		
	</node>

</launch>
